docs-user/guide-remote-profiling.md

```diff
@@ -24,3 +24,5 @@ Make any necessary adjustments in the presented options, like threads to sample
 # Tips
 
 * Avoid opening any other panels besides the Performance panel, to reduce the overhead of activated panels.
```

Comment:
I think this sentence should also be changed, because there is no Performance panel any more. And there is always going to be a selected panel (or the settings), right? Which panel should I select? Should I go to the settings before connecting to the device?

Reply:
Oh, I can't go to the settings before connecting. Heh. Oh, and if I follow the last tip, there is not going to be any selected panel, because the devtools thingy doesn't even appear. Great!